src/protocol/generate-python.py

```diff
@@ -76,11 +77,20 @@ def is_vector(payload):
 if not os.path.exists(output_path):
     os.makedirs(output_path)
 
+try:
+    os.chmod(os.path.join(output_path, "PingMessage.py"), stat.S_IWUSR | stat.S_IWGRP | stat.S_IWOTH)
```

Comment:
Create a variable that is `var = tat.S_IWUSR | stat.S_IWGRP | stat.S_IWOTH` and what this variables together does.